src/understanding-deis/concepts.md

```diff
@@ -11,27 +11,52 @@ We consider it an invaluable synthesis of much experience with
 software-as-a-service apps in the wild, especially on the
 Heroku platform.
 
-Deis is designed to run applications that adhere to [Twelve-Factor App][]
+Workflow is designed to run applications that adhere to [Twelve-Factor App][]
```

Comment:
Okay. Does that mean that eventually things like the titles "Understanding Deis" will eventually become "Understanding Workflow"? I feel like we should do this mass rename all in one fell swoop.